middleware/logger/logger.go

```diff
@@ -145,13 +178,15 @@ func New(config ...Config) fiber.Handler {
 		// Set latency start time
 		if cfg.enableLatency {
 			start = time.Now()
+			c.Context().SetUserValue("start", start.String())
```

Comment:
can you use more specific names for the beginning and the end, not that it conflicts with the values a user sets when configuring his app?

Reply:
GET